notifications.yaml

```diff
@@ -0,0 +1,220 @@
+        format: "uri"
+        description: "An absolute URI that identifies the problem type.  When dereferenced,\nit SHOULD provide human-readable documentation for the problem type\n(e.g., using HTML).\n"
+        default: "about:blank"
+        example: "https://example.com/problem/constraint-violation"
+      title: 
+        type: "string"
+        description: "A short, summary of the problem type. Written in english and readable\nfor engineers (usually not suited for non technical stakeholders and\nnot localized); example: Service Unavailable\n"
+      status: 
+        type: "integer"
+        format: "int32"
+        description: "The HTTP status code generated by the origin server for this occurrence\nof the problem.\n"
+        minimum: 100
+        maximum: 600
+        exclusiveMaximum: true
+        example: 503
+      detail: 
+        type: "string"
+        description: "A human readable explanation specific to this occurrence of the\nproblem.\n"
+        example: "Connection to database timed out"
+      instance: 
+        type: "string"
+        format: "uri"
+        description: "An absolute URI that identifies the specific occurrence of the problem.\nIt may or may not yield further information if dereferenced."
+  MessageContent: 
+    title: "MessageContent"
+    type: "object"
+    properties: 
+      title: 
+        type: "string"
+        minLength: 1
+        maxLength: 255
+        description: "The title of the message. At most 255 characters."
+      body_text: 
+        type: "string"
+        x-faker: "lorem.paragraph"
+      body_html: 
+        type: "string"
+        x-faker: "lorem.paragraph"
+    required: 
+      - "title"
+  MessageDefinition: 
+    title: "MessageDefinition"
+    type: "object"
+    properties: 
+      to: 
+        description: "One fiscal code (CF).\n"
+        type: "string"
+      dry_run: 
+        type: "boolean"
+        description: "This parameter, when set to true, allows developers to test a request without actually sending a message.\n\nThe default value is false."
+      time_to_live: 
+        type: "integer"
+        description: "This parameter specifies how long (in seconds) the message should be kept in storage if the device is offline. The maximum time to live supported is 1 year, and the default value is 1 hour."
+        minimum: 3600
+        maximum: 31536000
+      tags: 
+        type: "array"
+        default: []
+        items: 
+          type: "string"
+      content: 
+        $ref: "#/definitions/MessageContent"
+    required: 
+      - "to"
+      - "content"
+  MessageStatus: 
+    title: "MessageStatus"
+    type: "object"
+    properties: 
+      status: 
+        type: "string"
+        description: "ACCEPTED: the message has been accepted for delivery\n\nDELIVERED: the message was delivered to the channel\n\nFAILED: something wong happened, check the error code and message\n\nTIMED_OUT:  in case a TTL was set\n"
+        enum: 
```

Comment:
we can but this will prevent any code generator to apply any validation (or to generate a client that can handle the actual enum)